torch/_inductor/kernel/flex_attention.py

```diff
@@ -121,51 +66,43 @@ def build_subgraph_buffer(
         # expect that these are lifted inputs that fill up the '*other_buffers'
         # tuple and already have corresponding TensorBoxes passed in as args.
         if node.op == "placeholder":
-            is_lifted_input = cnt >= len(placeholder_inps)
-            lifted_input_index = index_to_other_buffers(cnt, graph_type)
-            env[node] = (
-                args[lifted_input_index] if is_lifted_input else placeholder_inps[cnt]
-            )
+            env[node] = args[cnt]
```

Comment:
I like this change, TLDR we made a mega list of [placeholders, then args] so that as we iterate we only get to the output node at the right arg count.

Can you update the docs, (remove args) and right some comment to the above effect.